Cython/Utility/ModuleSetupCode.c

```diff
@@ -109,6 +109,9 @@
   #define CYTHON_USE_DICT_VERSIONS 0
   #undef CYTHON_USE_EXC_INFO_STACK
   #define CYTHON_USE_EXC_INFO_STACK 0
+  #ifndef CYTHON_UPDATE_DESCRIPTOR_DOC
+    #define CYTHON_UPDATE_DESCRIPTOR_DOC 1
```

Comment:
The code was previously dependent on `CYTHON_COMPILING_IN_CPYTHON`, so let's not annoy other Python implementations with it.
```suggestion
    #define CYTHON_UPDATE_DESCRIPTOR_DOC 0
```